src/domain/infected-people/age-demographic/age-demographic-coordinates.ts

```diff
@@ -0,0 +1,170 @@
+import { localPoint } from '@visx/event';
+import { scaleBand, scaleLinear, ScaleTypeToD3Scale } from '@visx/scale';
+import { ScaleBand } from 'd3-scale';
+import { MouseEvent } from 'react';
+import {
+  GetTooltipCoordinates,
+  TooltipCoordinates,
+} from '~/components-styled/tooltip';
+import {
+  NationalInfectedAgeGroups,
+  NationalInfectedAgeGroupsValue,
+} from '~/types/data';
+import { AGE_GROUP_TOOLTIP_WIDTH } from './age-demographic-chart';
+
+export interface AgeDemographicCoordinates {
+  width: number;
+  height: number;
+  numTicks: number;
+  xMax: number;
+  yMax: number;
+  ageGroupPercentageScale: ValueOf<ScaleTypeToD3Scale<any, any, any>>;
+  infectedPercentageScale: ValueOf<ScaleTypeToD3Scale<any, any, any>>;
+  ageGroupRangeScale: ScaleBand<string>;
+  ageGroupPercentagePoint: (d: NationalInfectedAgeGroupsValue) => any;
+  infectedPercentagePoint: (d: NationalInfectedAgeGroupsValue) => any;
+  ageGroupRangePoint: (d: NationalInfectedAgeGroupsValue) => any;
+  getTooltipCoordinates: GetTooltipCoordinates<NationalInfectedAgeGroupsValue>;
+  isSmallScreen: boolean;
+  margin: {
+    top: number;
+    right: number;
+    bottom: number;
+    left: number;
+  };
+  values: NationalInfectedAgeGroupsValue[];
+  ageGroupRange: (d: NationalInfectedAgeGroupsValue) => string;
+  ageRangeAxisWidth: number;
+}
+
+export function getAgeDemographicCoordinates(
+  data: NationalInfectedAgeGroups,
+  isSmallScreen: boolean,
+  parentWidth: number
+): AgeDemographicCoordinates {
+  const values = data.values.sort((a, b) => {
+    const aStart = parseInt(a.age_group_range, 10);
+    const bStart = parseInt(b.age_group_range, 10);
+    return bStart - aStart;
```

Comment:
Sorting works directly on strings too 
```suggestion
    return bStart.localeCompare(aStart);
```